types/passport-jwt/package.json

```diff
@@ -23,10 +23,6 @@
             "name": "Alex Young",
             "githubUsername": "alsiola"
         },
-        {
-            "name": "David Ng",
```

Comment:
There are many David Ng accounts on GH, but cannot identify one that is obviously belonging to this individual

Reply:
@neviaumi Do you still want to be a maintainer of this package?